api/lib/infrastructure/serializers/csv/csv-serializer.js

```diff
@@ -166,6 +166,7 @@ function _getDataFromColumnNames({ csvLineKeys, headers, line }) {
         dateString: headerKeyInCurrentLine,
         inputFormat: 'DD/MM/YYYY',
         outputFormat: 'YYYY-MM-DD',
+        returnOriginalStringInCaseOfFailure: true,
       });
```

Comment:
On aurait aussi pu faire comme on le fait ailleurs (je ne sais plus où)
```suggestion
      }) ?? headerKeyInCurrentLine;
```